content/client-server-api/modules/reference_relations.md

```diff
@@ -0,0 +1,51 @@
+---
+type: module
+---
+
+### Reference relations
+
+{{% added-in v="1.4" %}}
+
+Generically referencing another event can be done with a `rel_type` of `m.reference`
+as a form of [relationship](#forming-relationships-between-events). There is no
+implied meaning behind the reference, and is usually context-dependent. Such an
+example is the [key verification framework](#key-verification-framework) which uses
+reference relations to associate distinct events with a specific verification attempt.
+
+When bundled, a reference relations appear as an object with a `chunk` field to
+indicate all the events which `m.reference` the event (the parent). Currently,
+only a single `event_id` field is present on the events in the `chunk`.
```

Comment:
There's also potentially a `next_batch` token to pass back into `/relations` to get more?

This seems to be at this discussion which was never resolved in the MSC: https://github.com/matrix-org/matrix-spec-proposals/pull/3267#discussion_r883541312

Reply:
I'd be inclined to intentionally punt that to a new MSC rather than backport it into MSC3267, I think. There's some questions about how it would work that start to feel like MSC-needing territory, like what if there's no more at the time the bundle is made or how long the server needs to keep the token valid for.

Reply:
Also, in practice clients *should* receive the follow-up events in most cases because they'll get the bundle from `/sync` and then be actively syncing again. There's a chance that they'll go offline again and create a gap, but backfill should repopulate the client's stores for local aggregation, or the event will be old enough where new references aren't necessarily important (for the current usecases in the spec, at least).

Reply:
That all sounds reasonable...but if we're not going to spec it, we should stop returning it from synapse (but I have no idea if clients are depending on it).

Reply:
I'd be comfortable removing it from Synapse tbh - it doesn't seem all that important to clients, but equally I am unsure if there's anyone out there using it (if they are, they can write an MSC imo)

Reply:
OK, we should file a Synapse issue at the very least then!